### DISCUSSION TITLE
ModuleNotFoundError: No module named 'customtkinter'

### DISCUSSION THREAD

**Discussion by SALAHZEMALI** [TomSchimansky/CustomTkinter#338] [General]
import customtkinter
ModuleNotFoundError: No module named 'customtkinter'

**Comment by PosthumasFrankestein** [comment]
Did you install `customtkinter`?

**Reply by SALAHZEMALI** [reply]
Yes

**Comment by felipetesc** [comment]
How many Python versions do you have ? Are you using pyenv, python 2 , python 3 ? What installation method you used? What's the current Pyhton version? What OS are you using ? What shell are you using ? Did you install venv ? What editor do you have ? Your code editor provides an automatical way to select your Python version?

Man, you need more information to help us to help you.
